packages/neon_framework/lib/src/widgets/app_bar.dart

```diff
@@ -19,136 +19,162 @@ import 'package:rxdart/rxdart.dart';
-                                  child: Text(
-                                    activeAppSnapshot.requireData.name(context),
-                                  ),
-                                ),
-                              ],
-                              if (appImplementations.hasError) ...[
-                                const SizedBox(
-                                  width: 8,
-                                ),
-                                NeonError(
-                                  appImplementations.error,
-                                  onRetry: appsBloc.refresh,
-                                  type: NeonErrorType.iconOnly,
-                                ),
-                              ],
-                              if (appImplementations.isLoading) ...[
-                                const SizedBox(
-                                  width: 8,
-                                ),
-                                Expanded(
-                                  child: NeonLinearProgressIndicator(
-                                    color: Theme.of(context).appBarTheme.foregroundColor,
-                                  ),
-                                ),
-                              ],
-                            ],
-                          ),
-                          if (accounts.length > 1) ...[
-                            Text(
-                              account.humanReadableID,
-                              style: Theme.of(context).textTheme.bodySmall,
-                            ),
-                          ],
-                        ],
-                      ),
-                actions: [
-                  if (unifiedSearchEnabled) ...[
-                    Flexible(
-                      child: SearchBar(
-                        focusNode: _searchBarFocusNode,
-                        hintText: NeonLocalizations.of(context).search,
-                        padding: const MaterialStatePropertyAll(EdgeInsets.only(left: 16)),
-                        onChanged: _searchTermController.add,
-                        trailing: [
-                          IconButton(
-                            onPressed: () {
-                              unifiedSearchBloc.disable();
-                            },
-                            tooltip: NeonLocalizations.of(context).searchCancel,
-                            icon: const Icon(
-                              Icons.close,
-                            ),
-                          ),
-                        ],
-                      ),
-                    ),
-                  ] else ...[
-                    const SearchIconButton(),
-                  ],
-                  const NotificationIconButton(),
-                  const AccountSwitcherButton(),
-                ],
-              );
+  Widget build(BuildContext context) {
+    assert(unifiedSearchBloc.enabled.value, 'The UnifiedSearchBloc must be enabled.');
```

Comment:
I don't think this is really necessary to check, even if it was not enabled it wouldn't do any harm.